covenant-signer/itest/containers/containers.go

```diff
@@ -0,0 +1,187 @@
+package containers
```

Comment:
Do we want same testing strategy as with vigilante/btc-staker/fp repo? Parallel e2e tests

Reply:
at some point definetly, though I would leave this for future improvements 👍